### ISSUE TITLE
Vertical alignment of Items in Header on iPhone X

### ISSUE THREAD

**Issue by alexisbronchart** [GeekyAnts/NativeBase#2128]
I'm using native-base to build the header of my app. Since I updated to version 2.7.1 the Items are displayed too high in the Header on iPhone X (see screenshot).

Here is how it looks on v2.7.1
<img width="516" alt="header" src="https://user-images.githubusercontent.com/1279527/42633135-cc17be22-85df-11e8-8054-7a8945b110e1.png">

Here is how I expect it to look (this is v2.3.9)
<img width="507" alt="previous" src="https://user-images.githubusercontent.com/1279527/42634507-aee2af3e-85e3-11e8-8c03-940e013207b0.png">

Notice how the search bar is now much closer to the notch.

To reproduce this just render the following view:
```
    <Container>
        <Header searchBar>
          <Button transparent>
            <Icon name="ios-people" />
          </Button>
          <Item>
            <Icon name="ios-search" />
            <Input placeholder="Search" />
          </Item>
          <Button transparent>
            <Text>Search</Text>
          </Button>
        </Header>
    </Container>
```

Versions:
node: 9.11.2
npm: 5.6.0
react-native: 0.55.4 & 0.56
react: 16.3.2 & 16.4.1
native-base version: 2.7.1

**Comment by akhil-geekyants** [comment]
@alexisbronchart if you have ejected theme, try changing the [`topInset` ](https://github.com/GeekyAnts/NativeBase/blob/master/src/theme/variables/platform.js#L269) value to 39

**Comment by alexisbronchart** [comment]
@akhil-geekyants  I thought about that too but the project from which I took the screenshots doesn't have an ejected theme.

**Comment by alexisbronchart** [comment]
I just tried again with the development branch, looks like it still doesn't work.

**Comment by SupriyaKalghatgi** [comment]
`development` branch do not have up-to-date transpiled code
either in package.json, modify main from `dist/src/index.js` to `src/index.js`
or transpile the code and then check

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.7.2](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.7.2)
